modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,124 @@
+= Configure Role-based Access Control
+:description: Role-based access controls provide an extension to ACLs for managing permissions at scale.
+:page-categories: Management, Security
+
+Role-based Access Control (RBAC) provides a way to configure permissions for provisioned users at scale. RBAC allows you to manage access for many users to many resources with a streamlined interface. This mechanism works in conjunction with all supported authentication methods.
+
+IMPORTANT: At present, Redpanda only supports RBAC for self-hosted clusters. Configuration is only supported through the command line using RPK.
+
+== RBAC
+
+RBAC is designed to address the critical challenge of access management at scale. It alleviates the process of manually maintaining and verifying a set of raw permissions (ACLs) for a user base that may contain thousands of users. RBAC provides you with a method to onboard new employees easier, audit accesses faster, and adapt more readily to evolutions of usage and compliance needs.
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to implement a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all principals with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as the allowed or denied principal respectively.
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
+
+* `user` - All authorized users have this role by default. There are no ACLs assigned to this role until you add them. This means users have no permissions to interact with any resources you add to your Redpanda cluster until you add relevant permissions to this or another role.
+* `admin` - By default, no users have this role. The default admin role provides full access to all resources in your Redpanda installation.
```

Comment:
```suggestion
* `admin` - By default, no users are assigned this role. The default admin role provides full access to all resources in your Redpanda installation.
```

Reply:
@oleiman So the admin role is for machines/apps to have access? Should we explain more explicitly than saying "all resources"?

Reply:
Ah--just saw your comment saying it is not going in the release...how would apps use/access RP without RBAC roles? ACLs only?

Reply:
Superuser access is available, as ever, for comprehensive "do anything" authorization. Superusers can then create roles, bind ACLs to those roles, assign user principals, etc.

Reply:
When I remove the 'Admin' role description, I'll add a note here that superuser access is required for establishing roles until users establish their own administrator roles.